packages/serverless/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@marblejs/serverless",
+  "version": "2.2.2",
+  "description": "Run Marble.js as a serverless function",
+  "keywords": [
+    "aws",
+    "lambda",
+    "serverless",
+    "marble.js",
+    "amazon",
+    "function"
+  ],
+  "author": "Krzysztof Miemiec <krzysztof.miemiec@gmail.com>",
+  "homepage": "https://github.com/marblejs/marble/tree/master/packages/serverless#readme",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@marblejs/core": "^2.2.2"
+  },
+  "devDependencies": {
+    "@marblejs/core": "^2.2.2",
+    "@types/aws-lambda": "^8.10.26",
```

Comment:
I'd rather not import it directly, since we can add other providers in the future. For example, if someone would like to use Google Cloud Functions, he doesn't need to install AWS typings. Also, I tried to load AWS module asynchronously, so when you bundle your app, you should only get code responsible for your preferred cloud provider, not all of them at once - which will save your execution time.